Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import os\r\nimport pygame\r\nfrom scripts.main_window import Tetris\r\n\r\nif __name__ == '__main__':\r\n    pygame.init()\r\n\r\n    pygame.display.set_caption('Тетрис')\r\n    fullname = os.path.join('data', 'icon.png')\r\n    image = pygame.image.load(fullname)\r\n    pygame.display.set_icon(image)\r\n    screensize = pygame.display.list_modes()\r\n    size = width, height = screensize[1][0], screensize[1][1]\r\n    screen = pygame.display.set_mode(size)\r\n    game = Tetris(screen)\r\n    game.start_game()\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision ea044ce82b754fed634c3e33cc283f29543a937f)
+++ b/main.py	(date 1704382081066)
@@ -13,4 +13,5 @@
     size = width, height = screensize[1][0], screensize[1][1]
     screen = pygame.display.set_mode(size)
     game = Tetris(screen)
+
     game.start_game()
